recipe/meta.yaml

```diff
@@ -25,15 +25,20 @@ requirements:
     - python
     - numpy >=1.16,<2
     - pandas >=1.0,<3
-    - pydantic >=1.7,<2
+    - pydantic >=1.7,<3
     - tqdm >=4.23,<5
     - toolz >=0.10,<1
     - typing-extensions >=4.0,<5  # [py<38]
-    # optional deps
+    # Optional dependencies that are part of the "torch" extra, 
+    # but the gluonts package we have on defaults right now includes 
+    # the "torch" extra by default in the runtime requirements. So we
+    # can't remove that dependency without breaking all our users.
     - pytorch >=1.9,<3
-    - pytorch-lightning >=1.5,<3
-    - protobuf
-    - scipy >=1.10,<2
+    - lightning >=2.0,<2.2
+    # Capping `lightning` does not cap `pytorch_lightning`, so we cap manually
+    - pytorch-lightning >=2.0,<2.2
+    - scipy >=1.7.3,<2  # [py<38]
```

Comment:
The requirements state that the scipy version be `~=1.73`, so this should say `>=1.7.3,==1.7.*` or at least `>=1.7.3,<1.8`: https://github.com/awslabs/gluonts/blob/v0.14.3/requirements/requirements-pytorch.txt#L6

Reply:
done, also for the other pinning of `scipy`, thank you @marcoesters